etc/krakend.json

```
@@ -127,25 +145,33 @@
           "roles": ["Registrar"],
           "jwk_local_path": "./etc/public_key.json",
           "disable_jwk_security": true,
-          "operation_debug": true
+          "operation_debug": true,
+          "propagate_claims": [
+            ["jti", "x-cwid"],
+            ["first_name", "x-first-name"],
+            ["last_name", "x-last-name"],
+            ["roles", "x-roles"]
+          ]
         }
       }
     },
     {
       "_comment": "Registrar 4: Deletes a specific class.",
-      "endpoint": "/api/classes/{id}",
+      "endpoint": "/api/classes/{class_term_slug}",
+      "input_headers": ["x-cwid", "x-first-name", "x-last-name", "x-roles"],
       "method": "DELETE",
       "backend": [
         {
-          "url_pattern": "/classes/{id}",
+          "url_pattern": "/classes/{class_term_slug}",
           "host": [
             "http://localhost:5100",
             "http://localhost:5101",
             "http://localhost:5102"
           ],
           "extra_config": {
             "backend/http": {
-              "return_error_code": true
+              "return_error_code": true,
+              "return_error_details": "enrollment_service"
```

```suggestion
              "return_error_code": true
```
Because we want the Gateway to forward the Error codes from the Backends, we need to removed this line.

When you use return_error_details, all status codes returned to the client are 200.
Read more: https://www.krakend.io/docs/backends/detailed-errors/#return-backend-errors-in-a-new-key